### PR TITLE
Visitor/User can View a Single Recipe 

### PR DESCRIPTION
**Pull request by yasmineezequiel**
[PT-link](https://www.pivotaltracker.com/story/show/169667227) 
## PR Content
In this PR we are adding a feature where the API returns a single recipe.
```
As a Visitor
In order to see recipe details
I would like an API that allows me to view a full recipe
```
## PR Contains:
- Testing of user/visitor can view single article spec.
- Show action for recipe controller.

### CODE REVIEW

File: app/controllers/recipes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class RecipesController < ApplicationController
+end
```
Comment (tochman) [inline app/controllers/recipes_controller.rb]: I was told that we should use a namespace. Like `V1` This controller should be namespaced if that is correct.

Comment (clalil) [reply app/controllers/recipes_controller.rb]: That is correct - we did agree on the namespace v1.

Comment (yasmineezequiel) [reply app/controllers/recipes_controller.rb]: Changed

**Review by beccaburns — APPROVED** [review]

**Review by miyeaier — APPROVED** [review]

**Review by shsteindorsson — APPROVED** [review]

**Review by beccaburns — APPROVED** [review]

**Review by clalil — APPROVED** [review]

**Review by oliverochman — APPROVED** [review]

File: app/controllers/v1/recipes_controller.rb [hunk 2]
```diff
@@ -17,9 +17,18 @@ def create
     end
   end
 
+  def show
+    recipe = Recipe.find(params[:id])
+    render json: recipe, serializer: Recipes::IndexSerializer
```
Comment (oliverochman) [inline app/controllers/v1/recipes_controller.rb]: You should use a separate serializer here, please create a `Recipes::ShowSerializer`

**Review by oliverochman — CHANGES_REQUESTED** [review]

**Review by oliverochman — APPROVED** [review]

**Pull request merged**